packages/exiftool.rb

```diff
@@ -0,0 +1,28 @@
+require 'package'
+
+class Exiftool < Package
+  description 'ExifTool is a platform-independent Perl library plus a command-line application for reading, writing and editing meta information in a wide variety of files.'
+  homepage 'https://owl.phy.queensu.ca/~phil/exiftool/'
+  version '11.71'
+  source_url 'https://owl.phy.queensu.ca/~phil/exiftool/Image-ExifTool-11.71.tar.gz'
+  source_sha256 '5258817bea57e0eb76d6783acb387f01bb5212ecb27e9db722f3e7f16cf0b5b5'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  @arch = 'x64'
+
+  depends_on 'perl'
+
+  def self.install
```

Comment:
You can build using the `Makefile.PL` like the Arch PKGBUILD does or since we include `cpanm` with our Perl maybe use:

```
cpanm http://cpan.metacpan.org/authors/id/E/EX/EXIFTOOL/Image-ExifTool-11.70.tar.gz
```